python/hsfs/training_dataset_feature.py

```diff
@@ -16,7 +16,7 @@
 
 import humps
 
-from hsfs.feature_group import FeatureGroup
+import hsfs.feature_group as feature_group
```

Comment:
yes otherwise import won't work properly because `FeatureGroup` is absent from `hsfs.feature_group` and `from hsfs.feature_group import FeatureGroup` causes error.